src/Example/UserRegistrationBundle/Transfer/UserTransfer.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/*
+ * Copyright (c) 2012-2014 KUBO Atsuhiro <kubo@iteman.jp>,
+ * All rights reserved.
+ *
+ * This file is part of PHPMentors_Training_Example_Symfony.
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the BSD 2-Clause License which accompanies this
+ * distribution, and is available at http://opensource.org/licenses/BSD-2-Clause
+ */
+
+namespace Example\UserRegistrationBundle\Transfer;
+
+use Example\UserRegistrationBundle\Entity\User;
+
+class UserTransfer
+{
+    /**
+     * @var string
+     */
+    private static $ACTIVATION_EMAIL_ACTIVATION_URI = 'https://www.example.org/users/registration/activation/';
+
+    /**
+     * @var array
+     */
+    private static $ACTIVATION_EMAIL_FROM = array('noreply@phpmentors.jp' => 'Exampleサービス');
+
+    /**
+     * @var string
+     */
+    private static $ACTIVATION_EMAIL_SUBJECT = 'Exampleサービス: ユーザー登録のご確認および完了手続きのご案内';
+
+    /**
+     * @var string
+     */
+    private static $ACTIVATION_EMAIL_TEMPLATE = 'ExampleUserRegistrationBundle:UserActivation:email.txt.twig';
+
+    /**
+     * @var \Swift_Mailer
+     */
+    private $mailer;
+
+    /**
+     * @var \Swift_Message
+     */
+    private $messageFactory;
+
+    /**
+     * @var \Twig_Environment
+     */
+    private $templateLoader;
+
+    /**
+     * @param \Swift_Mailer     $mailer
+     * @param \Swift_Message    $messageFactory
+     * @param \Twig_Environment $templateLoader
+     */
+    public function __construct(\Swift_Mailer $mailer, \Swift_Message $messageFactory, \Twig_Environment $templateLoader)
+    {
+        $this->mailer = $mailer;
+        $this->messageFactory = $messageFactory;
+        $this->templateLoader = $templateLoader;
+    }
+
+    /**
+     * @param  User    $user
+     * @return boolean
+     */
+    public function sendActivationEmail(User $user)
```

Comment:
__[development][phpmd]__
* The variable $ACTIVATION_EMAIL_FROM is not named in camelCase.
* The variable $ACTIVATION_EMAIL_SUBJECT is not named in camelCase.
* The variable $ACTIVATION_EMAIL_TEMPLATE is not named in camelCase.
* The variable $ACTIVATION_EMAIL_ACTIVATION_URI is not named in camelCase.